doc/man3/BIO_f_prefix.pod

```diff
@@ -0,0 +1,60 @@
+=pod
+
+=head1 NAME
+
+BIO_f_prefix, BIO_set_prefix, BIO_set_indent, BIO_get_indent
+- prefix BIO filter
+
+=head1 SYNOPSIS
+
+ #include <openssl/bio.h>
+
+ const BIO_METHOD *BIO_f_prefix(void);
+ long BIO_set_prefix(BIO *b, const char *prefix);
+ long BIO_set_indent(BIO *b, long indent);
+ long BIO_get_indent(BIO *b);
+
+=head1 DESCRIPTION
+
+BIO_f_cipher() returns the prefix BIO method. This is a filter for
+text output, where each line gets automatically prefixed and indented
+according to user input.
+
+BIO_set_prefix() sets the prefix to be used for future lines of text.
+This takes a copy of the prefix.
```

Comment:
s/takes/makes/  and also say that the default is an empty string?

Reply:
The default is NULL. Good point otherwise

Reply:
Done